### ISSUE TITLE
Instagram HACKED

### ISSUE THREAD

**Issue by trietnguye1** [ifedapoolarewaju/igdm#719]
My instagram immediately hacked(passsword/email/phone changed) after login this app. WARNING. DO NOT USE THIS APP!

**Comment by ifedapoolarewaju** [comment]
Hi there, so sorry to hear this. Where did you download the app from? As this is an open-source project, it’s possible you downloaded an edited version depending on where you downloaded it from.

**Comment by igorhorvacki** [comment]
MINE WAS TOO!! DO NOT USE THE APP!!

**Comment by ifedapoolarewaju** [comment]
@igorhorvacki again, I'll ask, where did you download the app from? was it downloaded from http://igdm.me

**Comment by igorhorvacki** [comment]
Yes it was on igdm.me and I someone logged in to my instagram account while
I WASNT getting the code 😑

sre, 21. nov 2018. 14.15 Ifedapo .A. Olarewaju <notifications@github.com>
је написао/ла:

**Comment by LordManhattan** [comment]
That makes zero sense. You probably have a system that's full of malware/viruses and that's how they hijacked your Instagram. Being open source, you're free to go through the code yourself, since it's literally published here on Github, but something tells me that won't happen.

**Comment by abduwalimuse** [comment]
@LordManhattan how do i go through the code? and is there a way to change it and download a version of the changed version just for myself?

**Comment by ifedapoolarewaju** [comment]
> Yes it was on igdm.me and I someone logged in to my instagram account while
I WASNT getting the code 😑

**Comment by ifedapoolarewaju** [comment]
@igorhorvacki could you please mail me a copy of the installation file which you downloaded from igdm.me

here's my email ifedapoolarewaju@gmail.com

> Yes it was on igdm.me and I someone logged in to my instagram account while
I WASNT getting the code 😑

**Comment by ackstorm23** [comment]
![igdm malwarew](https://user-images.githubusercontent.com/20215703/51208257-c93e1a80-18d2-11e9-905d-15d83861179d.png)

This binary appears to have malware in it.  I downloaded it 3 times from the https://igdm.me website.

**Comment by LordManhattan** [comment]
Interesting...

**Comment by ifedapoolarewaju** [comment]
@ackstorm23 thank you for sharing this. There is no adware or any sort of malware deliberately added to IGdm in itself.

While it is worth investigating other libraries that IGdm depends on, Please not that having your app detected as a malware on windows is a common issue with electron based apps https://github.com/electron/electron/issues/4485 (Which IGdm is).

Also IGdm Pro is an extension of IGdm so it is strange that it appears clean while IGdm shows this malware trait. However, this may happen be due to variation in electron version when the binary for the app is built

**Comment by Rahman2326** [comment]
> My instagram immediately hacked(passsword/email/phone changed) after login this app. WARNING. DO NOT USE THIS APP!

My Experience:
💗. Recovering Hacked WordPress Website
💗. Full Vulnerability Scan and WordPress website malware removal
💗. Vulnerability Assessment
💗. Antispam protection
💗. Analysis of database servers
💗. Restore a hacked website
💗. Ethical Hacking(White Hat Hacker)
💗. Hacked account recovery
💗. Password recovery
💗. Decode encrypted files
💗. Vulnerability Testing
💗....my WhatsApp number +8801314956025

**Comment by fuzzy76** [comment]
That an app is open source does not guarantee that the distributed binaries is directly compiled from the provided source, and doesn't have added malicious code.
